_docs/Contrib.md

```diff
@@ -1,5 +1,86 @@
+3) **Implementing keyboard support**: a set of keyboard behaviors implemented for the component. For each component type, the w3 organization has provided clear instructions on how to correctly implement them ([Accordion example](https://www.w3.org/TR/wai-aria-practices/#accordion)).
+4) **Focus**: in order for an interactive component to be supported by assistive technologies, its ability to gain focus is crucial. It is important for a composite component (i.e. dropdown, radio group) to gain focus through tab and to explicitly set the tabindex on its children. See [Keyboard navigation inside components](https://www.w3.org/TR/wai-aria-practices/#kbd_general_within) for an explanation on the correct use of tabindex in composite components.
+
+### Aria-prefixed Markup Attributes
+
+There are two types of aria-* attributes that need to be supported for a component:
+
+* User supplied attributes - are placed on a component by the developer who uses it in an application. All supported attributes need to be explicitly added in the props interface of the component (as optional) and are copied to the root element of the component using a decorator utility.
+
+    The current relevant attributes for all components are the text content attributes, used in interactive components, components with landmark roles and components with widget roles (see [explanation](https://www.paciellogroup.com/blog/2017/07/short-note-on-aria-label-aria-labelledby-and-aria-describedby/)). The attributes are aria-label, aria-labelledby, and aria-describedby.
+
+* Internal Implementation Attributes - are placed by the component code either on the root or on the relevant children in a composite component. Examples for these attributes are aria-autocomplete, aria-current, aria-checked and so on. The relevant attributes for a component will be added in the spec by the product manager.
+
+### Implementing Accessibility
+
+* Read the documentation for the component in the spec, using <a href="https://www.w3.org/TR/wai-aria-practices/" target="_blank">WAI-ARIA Practices</a> to fill in the details.
+* Use the decorator provided by Amir to copy the needed user-supplied attributes
+* Verify that you complete the above checklist (all 4 items are probably applicable to all components).
+* Verify that you use the general a11y contract test and write specific tests per component.
+
+    Note:
+    * When a component is not listed in the w3 draft, implement accessibility traits according to a similar component.
```

Comment:
I suggest going over the list of components and defining which is similar to which.